collectors/health.go

```diff
@@ -1285,6 +1285,10 @@ func (c *ClusterHealthCollector) collect(ch chan<- prometheus.Metric) error {
 		if state == "scrubbing" {
 			val -= *pgStateCounterMap["scrubbing+deep"]
 		}
+		if state == "snaptrim" {
+			val -= *pgStateCounterMap["snaptrim_wait"]
+		}
```

Comment:
Hmm, is this conflating `snaptrim` and `snaptrim_wait`?

Reply:
nm, I see what you're doing here. One day we'll tokenize this like adults.

Reply:
After a quick chat - tl;dr is no it's not :)